lighthouse-core/audits/byte-efficiency/uses-optimized-animated-images.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+/*
+ * @fileoverview Audit a page to ensure that videos are used instead of animated gifs
+ */
+'use strict';
+
+const Audit = require('../audit');
+const WebInspector = require('../../lib/web-inspector');
+
+// the threshold for the size of GIFs which we flag as unoptimized
+// see https://github.com/GoogleChrome/lighthouse/pull/4885#discussion_r178406623
+const GIF_BYTE_THRESHOLD = 100 * 1024;
+
+class UsesOptimizedAnimatedImages extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      name: 'uses-optimized-animated-images',
+      informative: true,
+      description: 'Use video formats for animated content',
+      helpText: 'Large GIFs are inefficient for delivering animated content. Consider using ' +
+        'MPEG4/WebM videos for animations and PNG/WebP for static images instead of GIF to save ' +
+        'network bytes. [Learn more](https://en.wikipedia.org/wiki/Video_alternative_to_GIF)',
```

Comment:
@paulirish @addyosmani any better link here? when does that post go live :)

Reply:
Looks like it wont be live within the next week.  But hopefully soon after that? cc @malchata

Reply:
Yep. That's the plan. I want to blast through feedback in the next couple days and get it into a PR. Knowing the PR process, it'd be next week.